js/apps/system/_admin/aardvark/APP/react/src/views/views/viewsList/ViewsTable.tsx

```diff
@@ -0,0 +1,83 @@
+import { Box, Link, Stack } from "@chakra-ui/react";
+import { createColumnHelper } from "@tanstack/react-table";
+import React from "react";
+import { Link as RouterLink, useHistory } from "react-router-dom";
+import { FiltersList } from "../../../components/table/FiltersList";
+import { ReactTable } from "../../../components/table/ReactTable";
+import { useSortableReactTable } from "../../../components/table/useSortableReactTable";
+import { LockableViewDescription, useFetchViews } from "../useFetchViews";
+
+const columnHelper = createColumnHelper<LockableViewDescription>();
+
+const TABLE_COLUMNS = [
+  columnHelper.accessor("name", {
+    header: "Name",
+    id: "name",
+    cell: info => {
+      const cellValue = info.cell.getValue();
+      return (
+        <Link
+          as={RouterLink}
+          to={`/views/${encodeURIComponent(cellValue)}`}
```

Comment:
There's a problem in react-router v5 which means this encoding doesn't really work & we end up with a malformed URL. To work around, we have to use a standard anchor link (`<Link>` from chakra) with an href, instead of router. You can use the [urlHelper -> createEncodedUrl function](https://github.com/arangodb/arangodb/blob/devel/js/apps/system/_admin/aardvark/APP/react/src/utils/urlHelper.ts#L1) function which I added recently for this purpose

currently this happens for eg (partially encoded): `http://localhost:3000/_db/_system/_admin/aardvark/index.html#views/sa-!%40%23%24%^%26*()`

<img width="1100" alt="image" src="https://github.com/arangodb/arangodb/assets/2976363/79485015-dff4-4d73-b920-04ace899275d">